server/lomas_server/utils/utils.py

```diff
@@ -0,0 +1,95 @@
+import io
```

Comment:
I think there was also a merge issue. These functions were moved to other parts of the code